### ISSUE TITLE
Mapper_parsing_exception skip on error object

### ISSUE THREAD

**Issue by ekevjn** [mongodb-labs/mongo-connector#778]
Hi everyone,
For many reasons, properties of documents in my mongo collection are not the same type so when i lunch mongo-connector for elastic, it throw  mapper_parsing_exception which cause by some of my documents had property which not match with elastic predefined mapping type then mongo connector stopped unexpectedly and shutting down. 
So my question is there is any option that tell mongo-connector to skip those errors documents and keep mapping others?
Any help would be greatly appreciated.

**Comment by holysky5** [comment]
yes ,I have the same question ,the mongo collector can not skip the last error op log entry
